client/src/modules/Schedule/components/StatusTabs/StatusTabs.test.tsx

```diff
@@ -0,0 +1,128 @@
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
What is the best practices in this repository to place tests for components? Please guide me

Reply:
@nataliagulko it's fine what you did. We have no guides. If you are ready to define some, please do and for example put it to contribution guide :)

Reply:
Not ready yet, I think. I saw that some other contributors created `__tests__` folders on components level. I would want to discuss it with other developers to get an agreement